.ghci

```diff
@@ -0,0 +1 @@
+:set -isrc/Cis194/Hw/Week1.hs -itest/Cis194/Hw
```

Comment:
I didn't know you could do this. That is awesome!

Reply:
Ah, this is another thing I tried using before I was able to run the spec file. IIRC the .ghci file has no impact when executing `stack runghc`.

Reply:
That is too bad. I wonder if stack has something like that. Like can I set a default command? I'll do some digging when I get a moment.